rules/windows/process_creation/proc_creation_win_susp_manageengine_pattern.yml

```
@@ -0,0 +1,47 @@
+title: Manage Engine Java Suspicious Sub Process
+id: cea2b7ea-792b-405f-95a1-b903ea06458f
+status: experimental
+description: Detects suspicious sub processes started by the Manage Engine ServiceDesk Plus Java web service process
+references:
+    - https://www.horizon3.ai/manageengine-cve-2022-47966-technical-deep-dive/
+    - https://github.com/horizon3ai/CVE-2022-47966/blob/main/CVE-2022-47966.py
```

We use permalinks just in case they change the structure
```suggestion
    - https://github.com/horizon3ai/CVE-2022-47966/blob/3a51c6b72ebbd87392babd955a8fbeaee2090b35/CVE-2022-47966.py
```